lib/Config.ts

```diff
@@ -14,7 +14,7 @@ class Config {
   public xudir: string;
   public db: DBConfig;
   public testDb: DBConfig;
-  public rpc: { host: string, port: number };
+  public rpc: { host:string, port: number };
```

Comment:
The standard style in our code looks like it has a space between the `:` and the type, so let's leave this line as it was.